apps/student/src/app/documents/[documentId]/page.tsx

```
@@ -0,0 +1,377 @@
+      <div className="mx-auto max-w-7xl px-6">
+        <div className={`grid gap-14 ${sourceType === "file" ? "lg:grid-cols-[1fr_480px]" : "grid-cols-1"}`}>
+          {/* Left Side: Form Info */}
+          <section className="pt-4">
+            <h1 className="font-bold text-[28px] text-gray-800 leading-none">
+              {title}
+            </h1>
+            <p className="mt-[26px] max-w-3xl text-[16px] text-gray-600 leading-[1.6]">
+              {description}
+            </p>
+            <div className="mt-4 flex items-center gap-2 text-[14px] text-gray-400">
+              <svg width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (2.4.15)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)